crates/anvil/src/config.rs

```diff
@@ -1032,6 +858,185 @@ latest block number: {latest_block}"
 
         backend
     }
+
+    pub async fn fork_db_setup(
```

Comment:
silly nit, but wdyt about naming this `setup_fork_db`? Also, let's add good docs to this! It's a pretty important function.